_includes/tutorials/rekeying-function/ksql/code/src/main/java/io/confluent/developer/RegexReplace.java

```diff
@@ -0,0 +1,17 @@
+package io.confluent.developer;
+
+import io.confluent.ksql.function.udf.Udf;
+import io.confluent.ksql.function.udf.UdfDescription;
+import io.confluent.ksql.function.udf.UdfParameter;
+
+@UdfDescription(name = "regexReplace", description = "Replace string using a regex")
+public class RegexReplace {
+
+  @Udf(description = "regexReplace string using a regex")
+  public String regexReplace(
```

Comment:
I should be testing for nulls in the parameters a la [Substring.java](https://github.com/confluentinc/ksql/blob/5.3.0-post/ksql-engine/src/main/java/io/confluent/ksql/function/udf/string/Substring.java#L62)